lru_cache/doubly_linked_list.py

```diff
@@ -48,26 +48,104 @@ def __init__(self, node=None):
     def __len__(self):
         return self.length
 
+    """Wraps the given value in a ListNode and inserts it 
+    as the new head of the list. Don't forget to handle 
+    the old head node's previous pointer accordingly."""
     def add_to_head(self, value):
-        pass
-
+        new_node = ListNode(value, None, self.head)
```

Comment:
Nice! Because the `ListNode` class has default args for `prev` and `next`, we don't have to include `None` here.